app/controllers/api/v1/reports_controller.rb

```diff
@@ -4,4 +4,29 @@ def features
 
     render json: feature_stats.to_json, status: :ok
   end
+
+  def selection_questions_summary
+    statistics = Reports::SelectionQuestionService.new.live_form_statistics
+
+    render json: statistics.to_json, status: :ok
+  end
+
+  def selection_questions_with_autocomplete
+    questions = Reports::SelectionQuestionService.new.live_form_pages_with_autocomplete
+
+    # adding the count as ActiveResource doesn't like parsing JSON with a single root key
+    render json: { questions:, count: questions.length }.to_json, status: :ok
```

Comment:
Do you reckon it's worth returning the count in the various `Reports::SelectionQuestionService` methods?

Reply:
Ah so just move where we construct the JSON response with the count into the service instead? Yeh I can do that.

I think my reasoning for putting it here was that we're only adding it due to a limitation of ActiveResource rather than actually needing it. So I thought putting it in the controller was slightly closer to the request and so the comment with the reasoning makes more sense here. But also don't feel too strongly.

Reply:
My thinking was that even if we're just doing it to make ActiveResource happy, it's a statistic that could make sense in the context of the SelectionQuestionService. But your logic makes sense too so I'm happy either way 😁

Reply:
Yeh ok, that reasoning is good enough for me - I'll move it

Reply:
Made the change